lib/assessment.js

```diff
@@ -13,14 +21,23 @@ var min = function min() {
 // and returns the sum of that set
 // assume that all the argument are numbers
 var sum = function sum() {
-
+  var total = arguments[0];
```

Comment:
The following loop will add all elements of `arguments` to `total`, including the 0th element. This causes the 0th element to be added to the sum twice.